src/Expensify.tsx

```diff
@@ -121,7 +121,12 @@ function Expensify() {
             return;
         }
 
-        ActiveClientManager.init();
+        // Set a timeout to initialize the client after a short delay.
+        // This is necessary because on Apple devices (web), Onyx events can be delayed,
+        // causing issues with client initialization if done immediately.
+        const isAppleDevice = /iPhone|iPad/.test(navigator.userAgent);
+        const isWeb = Platform.OS === 'web';
+        setTimeout(ActiveClientManager.init, isAppleDevice && isWeb ? 400 : 0);
```

Comment:
@sosek108 Let's use this existing method

```suggestion
        setTimeout(ActiveClientManager.init, isSafari ? 400 : 0);
```

Reply:
Done